search/ranker.cpp

```diff
@@ -541,8 +532,9 @@ void Ranker::ProcessSuggestions(vector<RankerResult> & vec) const
       GetSuggestion(r, m_params.m_query, m_params.m_tokens, m_params.m_prefix, suggestion);
       if (!suggestion.empty() && added < MAX_SUGGESTS_COUNT)
       {
-        // todo(@m) RankingInfo is not set here. Should it be?
-        if (m_emitter.AddResult(Result(MakeResult(r), suggestion)))
+        // todo(@m) RankingInfo is lost here. Should it be?
+        if (m_emitter.AddResult(Result(
+                MakeResult(r, true /* needAddress */, true /* needHighlighting */), suggestion)))
```

Comment:
Looks like we don't need address for suggestions.

Reply:
Let's remove it.
done